File: coco/src/main/java/coco/ep/m7i2/Sub.java
package coco.ep.m7i2;

import coco.ep.Exp;

public interface Sub<FT> extends coco.ep.m7.Sub<FT>, coco.ep.i1.Sub<FT>, coco.ep.m7i2.Exp<FT>, Factory<FT> {

    default Exp<FT> powby(Exp<FT> other) {
        return convert(this.lit(convert(this).eval())).powby(convert(other));
    }
}

File: coco_csharp/CoCo/EP/M7/Sub.cs
namespace EP.M7 {
    public interface Sub<FT> : Exp<FT>, EP.M6.Sub<FT> {
        EP.Exp<FT> Exp<FT>.powby(EP.Exp<FT> other) {
            return convert(lit(convert(this).eval())).powby(convert(other));
        }
    }
}